src/utils/response/resourceResponse.ts

```diff
@@ -0,0 +1,26 @@
+import { RawApiResponse } from './rawApiResponse';
+
+export type DeletedEntry = {
+  id: number;
```

Comment:
I believe this can be a string for some resources such as GeofenceToNode, but not sure

Reply:
Can we check this somehow? I hope it is not the case and if so we don't care at the moment.